### ISSUE TITLE
Create custom pages for 404 and 500 errors

### ISSUE THREAD

**Issue by ashmilhussain** [sirocco-ventures/raggenie#36]
Currently, there are no custom pages to handle 404 (Not Found) and 500 (Internal Server Error) errors, which can lead to a poor user experience.

Custom error pages should be implemented to improve user guidance and experience when encountering these errors.

<img width="832" alt="Screenshot 2024-09-29 at 5 33 10 PM" src="https://github.com/user-attachments/assets/4bff7c18-6637-47f9-ae7a-936cd41a59f2">

<img width="831" alt="Screenshot 2024-09-29 at 5 33 24 PM" src="https://github.com/user-attachments/assets/3235aeb1-2cad-467c-8cd4-fe802f19b65f">

figma fill be attched soon

**Comment by uchiha-vivek** [comment]
@ashmilhussain  How to get that 404 and 500 background , i could not see it in figma file .

**Comment by ashmilhussain** [comment]
hi @uchiha-vivek  , figma file is pinned on slack contributer page. please check

**Comment by agberoz** [comment]
@uchiha-vivek  any update on this ?

**Comment by uchiha-vivek** [comment]
@agberoz  yes sir working on it !

**Comment by sethumadhavan-k** [comment]
Hi @uchiha-vivek , any update on this ?

**Comment by uchiha-vivek** [comment]
@sethumadhavan-k  Sir 404 error page ui is made , just some minor css changes to do ! 

![Screenshot (100)](https://github.com/user-attachments/assets/76c4c4f7-04c8-46bb-9e51-644cb26867b4)

**Comment by sethumadhavan-k** [comment]
@uchiha-vivek any update on this ?

**Comment by uchiha-vivek** [comment]
@sethumadhavan-k  yes sir I made 404 page, just trying to handle css perfectly from figma. It's taking a bit time.

**Comment by agberoz** [comment]
@uchiha-vivek, are you designing the UI for the body? You can directly use the images

**Comment by sethumadhavan-k** [comment]
@uchiha-vivek any update on this?

**Comment by sethumadhavan-k** [comment]
@uchiha-vivek any update on this ?

**Comment by sethumadhavan-k** [comment]
@uchiha-vivek any update on this ?

**Comment by stablelane** [comment]
I would like to work on this issue.
